mri/reconstructors/utils/extract_sensitivity_maps.py

```diff
@@ -127,13 +143,19 @@ def get_Smaps(k_space, img_shape, samples, thresh,
             or len(thresh) != len(img_shape):
         raise NameError('The img_shape, max_samples, '
                         'min_samples and thresh must be of same length')
-    k_space, samples = \
+    k_space, *samples = \
```

Comment:
hmmm this is weird since `samples` might also have density compensation in it...
you can have `k_space, samples, *density_comp`, then do `if density_comp: density_comp = density_comp[0]`.

Reply:
Wait, does this work? I will check and update. We dont return anything sometimes

Reply:
Cool, this is good.

Reply:
yes if you don't return density compensation you will have `density_comp == []` as you found out.